templates/pkg/crd_sdk.go.tpl

```diff
@@ -192,17 +213,17 @@ func (rm *resourceManager) sdkCreate(
 	// the original Kubernetes object we passed to the function
 	ko := r.ko.DeepCopy()
 {{ $createCode }}
-{{ if $setOutputCustomMethodName := .CRD.SetOutputCustomMethodName .CRD.Ops.Create }}
-	// custom set output from response
-	rm.{{ $setOutputCustomMethodName }}(r, resp, ko)
-{{ end }}
 	if ko.Status.ACKResourceMetadata == nil {
 		ko.Status.ACKResourceMetadata = &ackv1alpha1.ResourceMetadata{}
 	}
 	if ko.Status.ACKResourceMetadata.OwnerAccountID == nil {
 		ko.Status.ACKResourceMetadata.OwnerAccountID = &rm.awsAccountID
 	}
 	ko.Status.Conditions = []*ackv1alpha1.Condition{}
+	{{ if $setOutputCustomMethodName := .CRD.SetOutputCustomMethodName .CRD.Ops.Create }}
+		// custom set output from response
+		rm.{{ $setOutputCustomMethodName }}(r, resp, ko)
+	{{ end }}
```

Comment:
the above changes are fine for this PR, but in the future it would be nice to place the ACKResourceMetadata and Conditions setting into a helper function called `setStatusDefaults()` or something like that to reduce the repetition :)